front/app/components/FilterBoxes/StatusFilter.tsx

```diff
@@ -3,7 +3,6 @@ import React, { memo, useCallback, MouseEvent } from 'react';
 import {
   fontSizes,
   colors,
```

Comment:
Used the new InputFilterCollapsible component in the existing Status filter just to make the UI cohesive for when we release this PR.

Any additional cleanup of existing code will happen when I work on the specific ticket :)